tests/test_gaap.py

```diff
@@ -188,8 +209,68 @@ def testGaapPluginForced(self, psfSigma):
         """
         self.runGaap(True, psfSigma)
 
+    def getFluxErrScaling(self, kernel, aperShape):
+        """Returns the value by which the standard error has to be scaled due
+           to noise correlations.
+
+           This is an alternative implementation to the `_getFluxErrScaling`
+           method of `BaseGaapFluxPlugin`, but is less effective.
 
+           Parameters
+           ----------
+           `kernel` : `lsst.afw.math.Kernel`
 
+        """
+        kim = afwImage.ImageD(kernel.getDimensions())
+        kernel.computeImage(kim, False)
+        weight = galsim.Image(np.zeros_like(kim.array))
+        aperSigma = aperShape.getDeterminantRadius()
+        gauss = galsim.Gaussian(sigma=aperSigma, flux=2*np.pi*aperSigma**2)
+        weight = gauss.drawImage(image=weight, scale=1.0, method='no_pixel')
+        kwarr = scipy.signal.convolve2d(weight.array, kim.array, boundary='fill')
+        fluxErrScaling = np.sqrt(np.sum(kwarr*kwarr))
+        fluxErrScaling /= np.sqrt(np.pi*aperSigma**2)
+        return fluxErrScaling
+
+    def testCorrelatedNoiseError(self, sigmas=[3.0, 4.0], scalingFactors=[1.15, 1.2, 1.25, 1.3, 1.4]):
+        """Test the scaling to standard error due to correlated noise.
+
+        The uncertainty estimate on GAaP fluxes is scaled by an amount
+        determined by the auto-correlation function of the PSF-matching kernel;
+        see Eqs. A11 & A17 of Kuijken et al. (2015). This test ensures that the
+        calculation of the scaling factors matches the analytical expression
+        when the PSF-matching kernel is a Gaussian.
```

Comment:
These comments should reference the new technote as well.

Reply:
The tech note is very fluid now, but I'll make it a point to reference the equations from there once it's a bit more stabilized. For now, referring to something static would be good.